src/tls.h

```diff
@@ -0,0 +1,20 @@
+#ifndef TLS_H
+#define TLS_H
+
+#define FQDN_LENGTH 256
+
+#define TLS_TYPE_HANDSHAKE_RECORD 22
+#define TLS_TYPE_APPLICATION_DATA 23
+
+#define TLS_EXTENSION_TYPE_SERVER_NAME 0
```

Comment:
I would make this private moving them back to the .c file. It's not a complete list of TLS types etc and probably not needed for now.